src/app/pages/send/send-crypto-asset-form/family/bitcoin/hooks/use-coin-selection.ts

```diff
@@ -0,0 +1,35 @@
+import { Money } from '@shared/models/money.model';
+
+import { useGetUtxosByAddressQuery } from '@app/query/bitcoin/address/utxos-by-address.query';
+import { useBitcoinFeeRatesInVbytes } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { useCurrentBtcAccountAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/bitcoin-account.hooks';
+
+import { coinselect } from '../coinselect/coinselect';
+
+// ts-unused-exports:disable-next-line
+export function useCoinSelection() {
+  const currentAccountBtcAddress = useCurrentBtcAccountAddressIndexZero();
+  const { data: utxos } = useGetUtxosByAddressQuery(currentAccountBtcAddress);
+  const { data: feeRate } = useBitcoinFeeRatesInVbytes();
+  return (sum: Money) => {
+    if (!utxos || !feeRate) return;
+
+    return {
+      high: coinselect(
+        utxos,
+        [{ address: currentAccountBtcAddress, value: sum.amount.toNumber() }],
+        feeRate.fastestFee
```

Comment:
Should this naming ref a time too, seems odd to switch from a time prediction to just `fastestFee`?

Reply:
Github is killing me! Nvm, I see that is coming from the new api now.